### DISCUSSION TITLE
Why is the new update asking for access to my browsing history?

### DISCUSSION THREAD

**Discussion by civiks** [typefully/minimal-twitter#132] [Q&A] [answered]
<img width="560" alt="SCR-20221218-p2e-2" src="https://user-images.githubusercontent.com/59787884/208298624-1cc7d867-4011-40e4-afd4-b92be9ce63f2.png">

**Comment by srid** [comment]
[`chrome.history`](https://developer.chrome.com/docs/extensions/reference/history/) is nowhere to be found in the source tree (all I could find was the addition of `chrome.tabs` in 3440649aa7511794f38871666a1286d6a17d0533) which leads me to believe that the extension published on the [Chrome store](https://chrome.google.com/webstore/detail/minimal-theme-for-twitter/pobhoodpcipjmedfenaigbeloiidbflp) corresponds to a source tree not on this repository.

**Comment by jiacai2050** [comment]
Also wonder why history permission is required for this extension, I can't imagine a scenario where this is required.

**Comment by bandit2077** [comment]
Bribed by the oligarchy?

**Comment by civiks** [comment]
> Bribed by the oligarchy?

That's probably the most logical explanation I could think of.

**Comment by srid** [comment]
Looks like this got addressed in 5.0.0.2

**Comment by ghost** [comment, marked as answer]
Hey thanks for your concern. The extension has been updated in 5.0.0.2 to remove the "tabs" permission, which caused this issue: https://github.com/typefully/minimal-twitter/commit/6d16ec5d1fd0f10f9f05d044d9ef2d842bfef09d. This was my fault, I thought it was required to update Twitter tabs on install: https://github.com/typefully/minimal-twitter/blob/main/background.js#L8, but turns out it was an unnecessary addition.

>This is the standard warning for the "tabs" permission. 
Source: https://stackoverflow.com/a/40660649/7948880

More info:
[![image](https://user-images.githubusercontent.com/41944255/208435900-947cb6c4-279f-4a78-a316-bc7dd2d00f89.png)](https://developer.chrome.com/docs/extensions/reference/tabs/#perms)
[![image](https://user-images.githubusercontent.com/41944255/208435915-13616c6d-0f11-43a1-beb6-f931436a0009.png)](https://stackoverflow.com/a/30395968/7948880)

**Reply by srid** [reply]
Thanks for the clarification, I didn't know that about the "tabs" permission.
